regparser/grammar/terms.py

```diff
@@ -26,10 +26,15 @@
     LineStart()
     + Optional(Suppress(unified.any_depth_p))
     + e_tag.setResultsName("head")
+    + Suppress(ZeroOrMore(unified.any_depth_p))
```

Comment:
But doesn't that mean that the tree isn't parsing correctly? By the time the definitions run, wouldn't the sub paragraph already be broken out?